Application/EdFi.Ods.Common/Database/Querying/Dialects/PostgreSqlDialect.cs

```diff
@@ -32,6 +32,12 @@ public override string GetLimitOffsetString(int? limit, int? offset)
 
         public override (string sql, object parameters) GetInClause(string columnName, string parameterName, IList values)
         {
+            // If list is empty, replace the IN clause with literal false condition
```

Comment:
Empty `IN` clauses are not valid in SQL, so we replace this with a constant-based expression that is false.